motor_driver.c

```diff
@@ -142,4 +175,40 @@ void motor_drv_init()
   delay_ms(1);
   MOTOR_RESET_PORT &= ~(1 << MOTOR_RESET_PIN);
   delay_ms(1);
+
+  /* Note that the X, Y and C motors are rated for
+  5A, but the gripper motor is only rated for 3A */
+  motor_drv_set_torque(XTABLE, TORQUE_VAL_5A);
+  motor_drv_set_isgain(XTABLE, FIVE);
+
+  motor_drv_set_torque(YTABLE, TORQUE_VAL_5A);
+  motor_drv_set_isgain(YTABLE, FIVE);
+
+  motor_drv_set_torque(CAROUSEL, TORQUE_VAL_5A);
+  motor_drv_set_isgain(CAROUSEL, FIVE);
+
+  motor_drv_set_torque(GRIPPER, TORQUE_VAL_3A);
+  motor_drv_set_isgain(GRIPPER, FIVE);
```

Comment:
Food for thought (no need to change right now), but perhaps we should make this configurable in the future.